js/__internal/grids/grid_core/sorting/m_sorting.ts

```diff
@@ -120,6 +120,7 @@ const HeaderPanelSortingExtender = extend({}, sortingMixin, {
         alignment: that.option('rtlEnabled') ? 'right' : 'left',
         allowSorting: groupColumn.allowSorting,
         sortOrder: groupColumn.sortOrder === 'desc' ? 'desc' : 'asc',
+        isGrouped: true,
```

Comment:
question: is it a new property of column object, or did we forget to add this property here?